src/Index/page.less

```diff
@@ -20,10 +20,56 @@
 
 .index-page__bar-chart {
   width: 770px;
+  padding: 0 20px 0 40px;
+}
+
+.index-page__slider-chart {
+  outline: none;
 }
 
 @media screen and (max-width: 1290px) {
```

Comment:
I think this line is causing a strange visual glitch when the screen width is >820px and <1290px, where the charts seem to shoot off to the right and disappear.